### DISCUSSION TITLE
Can't map more than one controller

### DISCUSSION THREAD

**Discussion by carlosmenmu** [darthcloud/BlueRetro#948] [Q&A]
I map successfully controls for my 8bitdo M30 controller, and when i try to map also myPS3 controller, it works but then the mapping of the 8bit controller is again back to stock.
I can't have both controllers with my custom mapping. It seems only one device saves the mapping.
Any help? Thanks!

**Comment by darthcloud** [comment]
BlueRetro only got one mapping.

They are not tied to any specific controller you are using.

**Reply by joaquincoelho** [reply]
Hi @darthcloud 

I hope you are well. I had the same question above, I've been trying to map two controllers to my gamecube adapter (just flashed beta24) which works great with the NSO gamecube controller.

So, if only one mapping is possible, what is the "select bluetooth device" under mapping config for? 

According to the documentation: "The Select Bluetooth device dropbox let you select the active Bluetooth input for which the setting will be saved to. You need to save for each modified device mapping setting." and that led me to believe that separate mappings for multiple controllers were possible.

**PS**: What I'm trying to do is to keep my NSO gamecube controller as is and use an 8bitdo SN30 as an SNES controller for GB games I run through the GB player...that would be awesome as the dpad on the gamecube controllers is tiny.

Thanks!

**Reply by darthcloud** [reply]
the different mapping are for multiplayer.

So player 1, pkayer 2 and so on
